caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -227,8 +232,15 @@ def predict_model(
                 The name of the request message to validate the model's task
             model_id (str):
                 The ID of the loaded model
+            input_streaming (bool):
```

Comment:
extremely minor nit: `input_streaming` is below `inference_func_name` in the list of arguments

Reply:
Good catch!